src/test/java/com/jalasoft/pivotal/steps/AccountSteps.java

```diff
@@ -0,0 +1,122 @@
+package com.jalasoft.pivotal.steps;
+
+import com.jalasoft.pivotal.core.Environment;
+import com.jalasoft.pivotal.pages.Header;
+import com.jalasoft.pivotal.pages.ProfileDropdown;
+import com.jalasoft.pivotal.pages.account.AccountModal;
+import com.jalasoft.pivotal.pages.account.AccountPage;
+import com.jalasoft.pivotal.pages.account.AccountSettingsPage;
+import com.jalasoft.pivotal.pages.account.SettingsPane;
+import com.jalasoft.pivotal.pages.project.*;
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import org.openqa.selenium.WebDriverException;
+import org.testng.Assert;
+
+import java.io.IOException;
+import java.util.Map;
+
+public class AccountSteps {
+
+    private ProfileDropdown profile;
+    private AccountPage accountsPage;
+    private AccountModal accountModal;
+    private AccountSettingsPage accountSettingsPage;
+    private SettingsPane settingsPane;
+    private ProjectMenu projectMenu;
+    private ProjectSettingsForm projectSettingsForm;
+
+    @And("I click Profiles")
+    public void iClickProfile(){
+        Header header = new Header();
+        profile = header.clickProfileDropdown();
+    }
+
+    @And("I click Accounts")
+    public void iClickAccountsInProfile(){
+        ProfileDropdown profile = new ProfileDropdown();
+        accountsPage = profile.clickAccounts();
+    }
+
+    @And ("I click Create Account")
+    public void iCreateAccount(){
+        AccountPage accounts = new AccountPage();
+        accountModal = accounts.clickCreateAccount();
+    }
+    @And ("I create new account {string}")
+    public void iSaveNewAccount(String accountName){
+        AccountModal accountModal = new AccountModal();
+        String account = Environment.getInstance().getValue(String.format("$['credentials']['%s']['name']", accountName));
```

Comment:
this information should came form feature file
not from environment.json

Reply:
do you mean that it is not necessary to save the "account name" in the environment.json? .. I put this info there, because I was using it several times in my feature file and if I would like to change this name, I will do it only one time if it is in environment.json. Please let me know your thoughts

Reply:
done